_drafts/oryx/feature-sets.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Feature sets
+description: Setup your frontend application using feature sets that contain all standard features
+last_updated: May 3, 2023
+template: concept-topic-template
+---
+
+A feature set is a group of related features that can be added to an Oryx application with a single reference. Using feature sets simplifies the process of setting up an application by reducing the amount of [boilerplate code](./boilerplate.md) required to configure and initialize the application.
+
+Feature sets are organized by business model, such as b2c, b2b, marketplace or fulfillment. The feature sets demonstrate the available features that Oryx provides. A feature list might be too opinionated for your business. You can therefor decide to use a complete feature set, or create your own sets.
+
+Each package provides its own features. For example, the product package exposes all product features available. This includes both components as well as the associated business logic.
+
+## Available Feature Sets
```

Comment:
If product feature considered a feature set, then this parahraph is about 'available business feature sets" (so we have two kinds of feature sets, business and... functional?)

Reply:
I went for "Application feature sets", but have made a reference in the intro to make that clear. 
Also improved a few other areas.